src/Controllers/PokemonController.php

```diff
@@ -82,13 +82,17 @@ public function capture(Request $request)
         $type = $request->get('type');
         $level = (int) $request->get('level');
 
-        $pokemon = $this->pokemonService->capture(
+        $this->pokemonService->capture(
             new PokemonId($uuid),
             new PokemonType($type),
             new PokemonLevel($level)
         );
 
-        return new JsonResponse($pokemon->toArray());
+        return new JsonResponse((new Pokemon(
```

Comment:
Ici aussi tu devrais ptet passer par `$this->pokemonLookup->get($uuid)->toArray()`